modules/botfun.py

```diff
@@ -6,7 +6,7 @@
 
 import random
 
-otherbot = "truncatedcone"
+otherbot = "quone"
```

Comment:
It might be good to make this a config option in the future, actually. There's no need to do it now, though.